src/psyclone/domain/lfric/lfric_constants.py

```diff
@@ -102,6 +111,8 @@ def __init__(self):
         LFRicConstants.VALID_STENCIL_DIRECTIONS = ["x_direction",
                                                    "y_direction"]
 
+        LFRicConstants.VALID_MESH_TYPES = ['gh_coarse', 'gh_fine']
```

Comment:
This file already has this entry (below).

Reply:
Oops.